tests/manager/api/admin/controller/test_collections.py

```diff
@@ -645,7 +646,10 @@ def test_collections_post_edit(
 
         library = db.default_library()
         collection2 = db.collection(name="Collection 2", protocol=OPDS2WithODLApi)
-        with flask_app_fixture.test_request_context_system_admin("/", method="POST"):
+        with (
+            flask_app_fixture.test_request_context_system_admin("/", method="POST"),
+            patch.object(opds_odl, "import_collection") as mock_import,
+        ):
```

Comment:
Since this collection has the `OPDS2WithODLApi` protocol, it tries to create a celery task to re-import the collection. We just mock that out to make sure it gets called.